tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/DevOpsService.cs

```diff
@@ -122,10 +125,10 @@ public async Task<ReleasePlan> GetReleasePlan(int releasePlanId)
             {
                 throw new Exception($"Failed to find release plan work item with release plan Id {releasePlanId}");
             }
-            return MapWorkItemToReleasePlan(releasePlanWorkItems[0]);
+            return await MapWorkItemToReleasePlan(releasePlanWorkItems[0]);
         }
 
-        private static ReleasePlan MapWorkItemToReleasePlan(WorkItem workItem)
+        private async Task<ReleasePlan> MapWorkItemToReleasePlan(WorkItem workItem)
```

Comment:
as this is now async, function name should be `MapWorkItemToReleasePlanAsync` to ensure people calling it are aware of it